pkgs/by-name/lo/loramon/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "loramon";
+  version = "0.9.7";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "markqvist";
+    repo = "LoRaMon";
+    rev = "refs/tags/${version}";
+    hash = "sha256-94tXhuAoaS1y/zGz63PPqOayRylGK0Ei2a6H4/BCB30";
+  };
+
```

Comment:
```suggestion
  nativeBuildInputs = with [
    setuptools
  ];

```

Reply:
I assume this should be `nativeBuildInputs = with python3.pkgs; [`, committed as such.